content/zh/docs/reference/config-api/kubelet-config.v1alpha1.md

```diff
@@ -0,0 +1,473 @@
+<tbody>
+<tr><td><code>name</code> <B>[Required]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <p>name is the required name of the credential provider. It must match the name of the
+provider executable as seen by the kubelet. The executable must be in the kubelet's
+bin directory (set by the --image-credential-provider-bin-dir flag).</p>
+</td>
+-->
+<table class="table">
+<thead><tr><th width="30%">字段</th><th>描述</th></tr></thead>
+<tbody>
+<tr><td><code>name</code> <B>[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <p>name 是凭据提供程序的必需名称。它必须与 Kubelet 看到的提供程序的可执行文件名称相匹配。
+   该可执行文件必须位于 Kubelet 的 bin 目录中（由 --image-credential-provider-bin-dir 标志设置）。</p>
+</td>
+
+<!--
+</tr>
+<tr><td><code>matchImages</code> <B>[Required]</B><br/>
+<code>[]string</code>
+</td>
+<td>
+   <p>matchImages is a required list of strings used to match against images in order to
+determine if this provider should be invoked. If one of the strings matches the
+requested image from the kubelet, the plugin will be invoked and given a chance
+to provide credentials. Images are expected to contain the registry domain
+and URL path.</p>
+-->
+</tr>
+<tr><td><code>matchImages</code> <B>[必需]</B><br/>
+<code>[]string</code>
+</td>
+<td>
+   <p>matchImages 是一个必填的字符串列表，用于匹配镜像以确定是否应调用此提供程序。
+如果其中一个字符串与来自 Kubelet 的请求镜像匹配，则将调用插件并有机会提供凭据。
+镜像应包含 Registry 域和 URL 路径。</p>
+
+<!--
+<p>Each entry in matchImages is a pattern which can optionally contain a port and a path.
+Globs can be used in the domain, but not in the port or the path. Globs are supported
+as subdomains like '<em>.k8s.io' or 'k8s.</em>.io', and top-level-domains such as 'k8s.<em>'.
+Matching partial subdomains like 'app</em>.k8s.io' is also supported. Each glob can only match
+a single subdomain segment, so *.io does not match *.k8s.io.</p>
+<p>A match exists between an image and a matchImage when all of the below are true:</p>
+-->
+<p>matchImages 中的每个条目都是一个模式，可以选择包含一个端口和一个路径。
+Glob 可以在域中使用，但不能在端口或路径中使用。
+支持 Glob 作为子域，如 “<em>.k8s.io” 或 “k8s.</em>.io”，以及顶级域，如 “k8s.<em>”。
+支持匹配部分子域，如 “app</em>.k8s.io”。每个 Glob 只能匹配单个子域段，因此 *.io 不匹配 *.k8s.io。</p>
+<p>当以下所有条件都为真时，镜像和 matchImage 之间存在匹配：</p>
+
+<!--
+<ul>
+<li>Both contain the same number of domain parts and each part matches.</li>
+<li>The URL path of an imageMatch must be a prefix of the target image URL path.</li>
+<li>If the imageMatch contains a port, then the port must match in the image as well.</li>
+</ul>
+<p>Example values of matchImages:</p>
+<ul>
+<li>123456789.dkr.ecr.us-east-1.amazonaws.com</li>
+<li>*.azurecr.io</li>
+<li>gcr.io</li>
+<li><em>.</em>.registry.io</li>
+<li>registry.io:8080/path</li>
+</ul>
+</td>
+</tr>
+-->
+<ul>
+<li>两者都包含相同数量的域部分，并且每个部分都匹配。</li>
+<li>imageMatch 的 URL 路径必须是目标镜像 URL 路径的前缀。</li>
+<li>如果 imageMatch 包含一个端口，那么该端口也必须与镜像匹配。</li>
+</ul>
+<p>matchImages 的示例值：</p>
+<ul>
+<li>123456789.dkr.ecr.us-east-1.amazonaws.com</li>
+<li>*.azurecr.io</li>
+<li>gcr.io</li>
+<li><em>.</em>.registry.io</li>
```

Comment:
```suggestion
<li>*.registry.io</li>
```

Reply:
What is the reason for the modification here?

Reply:
This looks like a generator bug.